packages/logger/src/config/EnvironmentVariablesService.ts

```diff
@@ -123,7 +120,7 @@ class EnvironmentVariablesService extends ConfigService {
    * @returns {string}
    */
   public getXrayTraceId(): string {
-    return this.get(this.xRayTraceIdVariable);
+    return super.getXrayTraceId();
```

Comment:
Given that this class extends commons' `EnvironmentVariablesService` and that the commons superclass marks `getXrayTraceId()` as public, do we need to define a `getXrayTraceId()` method here?

Reply:
Very good spot @misterjoshua, I missed this. 

I have also removed the extra `getServiceName` one from all.